tests/sheet/sheet_manipulation_plugin.test.ts

```diff
@@ -315,133 +314,14 @@ describe("Columns", () => {
     });
   });
 
-  describe("Correctly update borders", () => {
```

Comment:
can't we keep those tests ?
They are super annoying and hard to read and understand, but it makes it quite risky to delete them because we don't know what they test (a lot, at least indirectly)

Reply:
Exactly why I want to remove them, As far as I can see: these test the border continuity (ex adding a row after a cell with border, between 2 cells with borders,...) but they are so hard to read that you cannot understand without writing the test on a sheet of paper.

This behaviour is already thoroughly tested in border_plugin.test